application/listeners/listeners.go

```diff
@@ -0,0 +1,71 @@
+package listeners
+
+import (
+	"github.com/gobuffalo/events"
+	"github.com/silinternational/wecarry-api/domain"
+	"github.com/silinternational/wecarry-api/models"
+	"time"
+)
+
+// For new listener functions, register them at the end of the
```

Comment:
Why not just put the slice here?

Reply:
Ha, ha.  That's my Python experience showing through.